PIL/Image.py

```diff
@@ -1122,6 +1186,7 @@ def filter(self, filter):
         return merge(self.mode, ims)
 
     def getbands(self):
+        # type: () -> Tuple[Any]
```

Comment:
Should return a tuple of single character strings, where len==number of bands/channels.